solution.py

```diff
@@ -584,19 +687,15 @@ def cm_analysis(y_true, y_pred, title, figsize=(10,10)):
 # %% [markdown]
 # <div class="alert alert-info"><h4>
 # Task 3.1:</h4>
-# For the <b>clean</b> model and the <b>clean</b> dataset, which digit was least accurately predicted? What did the model predict instead? Why do you think these digits were confused by the model?
+# For the <b>clean</b> model and the <b>clean</b> dataset,
+# which digit was least accurately predicted? What did the model predict instead? Why do you think these digits were confused by the model?
 # </div>
 
 # %% [markdown] tags=["solution"]
 # **3.1 Answer:**
 #
-# The clean model on the clean dataset predicted 5s least accurately, with some confusion with 6s and 3s. These are likely confused by the model as handwritten 5s may look like 6s (almost closed bottom part) or 3s (presence of 3 horizontal segments).
-
-# %% [markdown] tags=["solution"]
-# **3.1 Answer from 2023 Students**
-#
-# 5 is the least accurately predicted digit. It is most confused with 6 or 3.
-# Handwriting creates fives that look like sixes or threes.
+# The clean model on the clean dataset predicted 5s least accurately, with some confusion with 6s and 3s.
```

Comment:
My clean model confused 4/9 and 7/9 most - presumably this is due to randomness, so perhaps we should edit the solution to say that their model might be different.